public/landing/styles.css

```
@@ -1,381 +1,581 @@
+  --paper-dim: #b9b2a5;
+  --paper-faint: #6e6a7a;
+
+  --verm: #ff4d2e;
+  --verm-deep: #c9331f;
+  --verm-soft: rgba(255, 77, 46, 0.14);
+  --violet: #8f7ff0;
+  --violet-soft: rgba(143, 127, 240, 0.14);
+  --green: #2fd48f;
+  --green-soft: rgba(47, 212, 143, 0.14);
+
+  --font-display: 'Fraunces', Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)